controller/access/access.go

```diff
@@ -772,6 +772,8 @@ func CompileUriPermitsMapping() {
 				"v1/system/license",
 				"v1/system/summary",
 				"v1/internal/system",
+				"v1/remote_export_repository",
```

Comment:
this remote repository could be used for import in the future as well.
can we use "remote_repository" instead of "remote_export_repository"?

Reply:
Good point. I have changed the verbiage everywhere to just `remote repository` instead of `remote export repository`.